fiftyone/operators/store/transient_jobs.py

```
@@ -0,0 +1,256 @@
+        def finish(job, now):
+            if not _is_current_worker(job, fencing_token, now):
+                return None
+            job["state"] = state
+            job["lease_expires_at"] = None
+            job["result"] = result
+            job["error"] = error
+            return job
+
+        return self._mutate(job_id, finish)
+
+    def _mutate(self, job_id, transform):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_mutate`

(ANN202)